src/axom/sidre/core/DataStore.hpp

```diff
@@ -98,6 +98,28 @@ class DataStore
   /// \brief Return true if DataStore owns a Buffer with given index; else false
   bool hasBuffer(IndexType idx) const;
 
+  /*!
+   * \brief Return information about DataStore Buffers in fields of given
+   *        Conduit Node.
```

Comment:
I'd suggest "Inserts information" to avoid confusion with the concept of a return type.

Reply:
Agreed. That would be clearer. I will push a change.